qiskit/assembler/assemble_schedules.py

```diff
@@ -189,7 +189,7 @@ def _assemble_instructions(
                 instruction.channels[0], name=name)
 
         if (isinstance(instruction, instructions.Play) and
-                isinstance(instruction.pulse, (library.SamplePulse, library.Waveform))):
```

Comment:
It's not important, cause it will get fixed when SamplePulse is removed, but this does show up elsewhere if you want to do a find-all